test/e2e/angular/font-setting-test-ng.js

```diff
@@ -30,7 +30,7 @@
         it("should load all fonts", function() {
           element(by.css("#font1  .mce-btn[aria-label='Font Family']")).click();
           element.all(by.css("#mceu_42-body div")).then(function(elements) {
-            expect(elements.length).to.equal(888);
+            expect(elements.length).to.equal(889);
```

Comment:
why not change this to
to.be.above(800)
to avoid constant changes in the future every time the external package adds new fonts ?

Reply:
@santiagonoguez good point, I will update that. I had made that change before you updated the test for the text widget. Thanks!